runtime/near-vm-runner/src/cache.rs

```diff
@@ -35,3 +43,89 @@ pub(crate) fn compile_module(
     let prepared_code = prepare::prepare_contract(code, config)?;
     wasmer_runtime::compile(&prepared_code).map_err(|err| err.into_vm_error())
 }
+
+fn get_cached_name(code_hash: &Vec<u8>) -> String {
+    // For a prototype we store compiled contracts in file, later on we'll use database.
```

Comment:
Now that database is used, should this be removed?

Reply:
Database is not yet implemented.